docs/0-RefGuide.adoc

```diff
@@ -1,12 +1,11 @@
 ////
-  Copyright (c) 2022 Goldman Sachs and others. All rights reserved. 
-  This program and the accompanying materials   are made available 
-  under the terms of the Eclipse Public License v1.0 and 
-  Eclipse Distribution License v. 1.0 which accompany this distribution.
-  The Eclipse Public License is available at 
-  http://www.eclipse.org/legal/epl-v10.html.
-  The Eclipse Distribution License is available at 
-  http://www.eclipse.org/org/documents/edl-v10.php.
+   Copyright (c) 2022 The Bank of New York Mellon.
```

Comment:
The original copyright was correct as some of the content was from the original markdown version of the guide.